sdk/cxx/src/transaction_processor.cpp

```diff
@@ -88,11 +88,12 @@ void TransactionProcessor::HandleProcessingRequest(const void* msg,
         request.ParseFromArray(msg, msg_size);
 
         TransactionHeader* txn_header(request.release_header());
+        TxHeaderPtr txHeader(new TxHeaderWrapper(txn_header)); // YYY TBD
```

Comment:
What does YYY TBD mean?

Reply:
My initials

Reply:
We don't allow TODO comments in the code base - they should be reflected in tasks on the Jira board.  Likewise, a comment like that gives no context to its meaning to anyone other than the original commenter. E.g. what is TBD on that line?